heureka/ui/src/components/components/ComponentsListItem.jsx

```diff
@@ -6,12 +6,20 @@
 import React from "react"
 import { DataGridRow, DataGridCell } from "@cloudoperators/juno-ui-components"
 
+const sumTotalInstances = (versions) => {
+  let sum = 0
+  versions.forEach((v) => (sum += v?.node?.componentInstances?.totalCount))
+  return sum
+}
 const ComponentsListItem = ({ item }) => {
   return (
     <DataGridRow>
       <DataGridCell>{item?.node?.name}</DataGridCell>
       <DataGridCell>{item?.node?.type}</DataGridCell>
       <DataGridCell>{item?.node?.componentVersions?.totalCount}</DataGridCell>
+      <DataGridCell>
+        {sumTotalInstances(item?.node?.componentVersions?.edges)}
```

Comment:
Why are you not reading the totalCount of componentInstances directly from `item?.node?.componentInstances?.totalCount `?